portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,26 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+    <nav>
+      <strong>Nathan Safir</strong>
+      <ul> 
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="projects.html">Projects</a></li>
+      </ul>
+    <br />
+    </nav>
+    <img src=/images/me.jpg>
+    <p>Student at the University of Georiga (Class of 2022)</p>
+    <p>B.S./M.S. Computer Science and Artificial Intelligence, B.A. Geography</p>
+    <br />
+    <p><a href="mailto:nssafir@gmail.com">nssafir@gmail.com</a></p>
+    <br />
+    <p><a href="https://github.com/nssafir/">Github</a></p>
+    <p><a href="https://www.linkedin.com/in/nathan-safir">Linkedin</a></p>
   </body>
-</html>
+</html>
```

Comment:
This file is missing a final newline. Try using the autoformat in the future maybe, or just add a carriage return for now.